dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+QT_MINIMAL="5.9"
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay \
+		https://www.kdab.com/?s=gammaray"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT=0
+IUSE="3d auto-detect backtrace bluetooth +client \
+	concurrent designer -doc feedback geolocation +gpl-only-features \
```

Comment:
`feedback` does not need any extra dependency, is there any good other reason to not just build it unconditionally? Furthermore, in CMakeLists.txt we find out that it basically enables a bundled version of `kuserfeedback` inside `3rdparty`. We don't like bundled libs, maybe just hard-disable it until we figure out a way to use an external package kuserfeedback (and actually package that)? ;)